### PR TITLE
Add case studies section and seed with primary wayfair tremor use cases

### PR DESCRIPTION
**Pull request by darach**
Signed-off-by: Darach Ennis <dennis@wayfair.com>

### CODE REVIEW

File: community/case-studies/wayfair/data-distribution.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+-   Classification, Categorization and rate limiting for Traffic Shaping
+
+-   Enrichment, Normalization and micro-format parsing
+
+-   Structural pattern matching over hierarchical nested JSON-like data
+
+-   The ability to patch, merge and iterate over JSON-like data
+
+-   A familiar expression language like in regular programming languages
+    > with arithmetic, multiplicative and other common operations on
```
Comment (Licenser) [inline community/case-studies/wayfair/data-distribution.md:119]: I think the `>` needs to go to properly render this (same below)

File: community/case-studies/wayfair/data-distribution.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+Instead of our operations and service reliability teams needing to learn
+and manage the service deployment and administrative lifecycle of
+multiple components in this evolution we displace and replace those
+systems - typically operating as sidecars on source systems where our
+applications run on virtual machines, bare metal or containerized
+environments with tremor.
+
+The relatively primitive domain specific language developed as part of
+the initial solution for traffic shaping is replaced with a new domain
+specific language \`tremor-script\` designed for expressive data
```
Comment (Licenser) [inline community/case-studies/wayfair/data-distribution.md:107]: there are a number of places where gdocs seems to have escaped the ticks:
```
\`tremor-script\`
```

should probably be:
```
`tremor-script`
```

(same in a few more places, probably search&replaceable :D)

File: community/case-studies/wayfair/data-distribution.md (hunk 1 above)
Comment (darach) [reply community/case-studies/wayfair/data-distribution.md]: Well spotted - those are injected by `pandoc` - thats one i missed 👍

File: community/case-studies/wayfair/data-distribution.md (hunk 2 above)
Comment (darach) [reply community/case-studies/wayfair/data-distribution.md]: Another `pandoc` artefact!

**Comment by Licenser** [comment]
Could we move the case studies into blog posts? That was one of Celeste's suggestions and it prevents adding a new section that clutters up the page. One of the problems we had with the webpage's first version was too many things in too many places adding sections for everything new is the path back to that I fear :(

File: community/case-studies/wayfair/000_index.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
+# Wayfair
+
+![wayfair](./media/wayfair.png)
+
+Development on tremor started in the autumn of 2018 as a point solution
+for introducing a configurable traffic shaping mechanism for our
+infrastructure platform engineering organization.
+
+There were two use cases being considered at the time - this use case
```
Comment (skoech) [inline community/case-studies/wayfair/000_index.md:9]: I find this phrasing a little confusing. How about something like, "There were two use cases being considered at the time -  the traffic-shaping use case, which had the highest priority, and the distributed data replication use case for our Kafka-based environments."

File: community/case-studies/wayfair/000_index.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+where US folk go peak shopping! ) - and it worked flawlessly.
+
+Although designed with our logging systems in mind, our metrics systems
+team saw that the same system would work for their environment (
+InfluxDB downstream rather than ElasticSearch, but the logic was
+basically the same ).
+
+Tremor was designed to detect or sense when downstream systems are
+suffering back-pressure and to intelligently - based on user defined
+logic - to react by adaptively tuning the data distribution to
```
Comment (skoech) [inline community/case-studies/wayfair/000_index.md:43]: There's an extra "to".
... and to intelligently- based on user defined
logic- react

File: community/case-studies/wayfair/000_index.md (hunk 3 above)
Comment (darach) [reply community/case-studies/wayfair/000_index.md:9]: Agreed, added your rephasing!

File: blog/2021-11-09-php-dead-code-detection.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Dead Code Detection in PHP
```
Comment (Licenser) [inline blog/2021-11-09-php-dead-code-detection.md:1]: All the blog posts need blog headers here is an example:
```
---
title: Tremor is now Open Source!
author: The Tremor Team
author_url: https://github.com/tremor-rs
author_image_url: https://avatars.githubusercontent.com/u/60009416?s=200&v=4
tags:  [hello, welcome, announcement, open-source]
image: /img/logo-minimalism.png
draft: false
hide_table_of_contents: false
description: Announcement of Tremor as an OSS project.
---
```

**Review by Licenser — CHANGES_REQUESTED** [review]

**Comment by Licenser** [comment]
I would add different dates to the files otherwise they'll show up out of order, and add the tag `casestudy` to them? that way that tag can become a collection of all case studies

**Review by Licenser — APPROVED** [review]
:rocket:

**Pull request merged**